dracut/module-setup.sh

```diff
@@ -6,4 +6,11 @@ install() {
 
     # Don't install bash
     > "${initdir}/usr/bin/bash"
+
+    # Don't install vi and various deps
+    > "${initdir}/usr/libexec/vi"
+    > "/lib64/libm.so.*"
+    > "/lib64/libacl.so.*"
+    > "/lib64/libattr.so.*"
```

Comment:
I'd rather not skip the libs there is a large likelyhood that e.g. ostree depends on it

Reply:
Yeah, these are not big anyway will remove. vi is kinda meaty though